pandas/tests/window/test_rolling.py

```diff
@@ -498,7 +498,7 @@ def test_rolling_count_default_min_periods_with_null_values(constructor):
                 ({"A": [2, 3], "B": [5, 6]}, [1, 2]),
             ],
             2,
-            3,
+            2,
```

Comment:
Related to the whatsnew entry: This should not have passed previously.

Reply:
what whatsnew entry?

Reply:
Oops, though I added it is previously. Should be there now.